share/initialization/default

```diff
@@ -1,6 +1,6 @@
 ##### Default Yashrc #####
 # This file is automatically loaded when yash starts up and a user-specific
-# initialization script cannot be read from ~/.yashrc.
+# initialization script cannot be read from $XDG_CONFIG_HOME/yash/rc or ~/.yashrc.
```

Comment:
Should I include the fallback path in documentation too or only reference the XDG path?